about.md

```diff
@@ -94,3 +94,23 @@ DELETE https://<host>:<port>/<index-name>/_doc/<document-id>
 ```
 
 You can change most OpenSearch settings using the REST API, modify indexes, check the health of the cluster, get statistics---almost everything.
+
+## Advanced concepts
+
+The following section describes more advanced OpenSearch concepts.
+
+### Translog
+
+Any index changes, such as indexing or deleting documents, are written to disk during a Lucene commit. However, Lucene commits are expensive operations, so they cannot be performed after every change to the index. Instead, each shard records every indexing operation in a transaction log called translog. When a document is indexed, it is added to the memory buffer and recorded in the translog. Frequent refresh operations write the documents in the memory buffer to a segment and then clear the memory buffer. Periodically, a flush performs a Lucene commit, which includes writing the segments to disk using `fsync`, purging the old translog, and starting a new translog. Thus, a translog contains all operations that have not yet been flushed.
```

Comment:
> When a document is indexed, it is added to the memory buffer and recorded in the translog.

Maybe it's kind of implied, but nothing explicitly mentions that the translog is durable but the "in memory buffer" is not. A naive reading of this doesn't explain why the document is recorded in both places. (The answer is that the translog is durably written to disk and can survive a process or host restart, but any data in the in memory buffer will be lost.)